wsmaster/che-core-api-workspace/src/main/resources/schema/devfile.json

```diff
@@ -279,7 +279,11 @@
                       "{\"java.home\": \"/home/user/jdk11\", \"java.jdt.ls.vmargs\": \"-Xmx1G\"}"
                     ],
                     "additionalProperties": {
-                      "type": "string"
+                      "type": [
+                        "boolean",
+                        "string",
+                        "number"
```

Comment:
Note than for json-schema draft 7, `"number"` types include floating point numbers. Can this cause an issue if e.g. a float is stored in additional properties and then de-serialized through `PreferencesDeserializer`? It looks like it would fall through to the default case.

Since json-schema is pretty [iffy](https://json-schema.org/understanding-json-schema/reference/numeric.html#integer) with specifying integers, maybe it makes sense to handle a floating point case as well in the deserializer?

Reply:
Ok